test/csuite/random_directio/main.c

```diff
@@ -814,26 +808,20 @@ check_schema(WT_SESSION *session, uint64_t lastid, uint32_t threadid, uint32_t f
             check_one_entry(session, uri, uri, uri);
     }
     if (has_schema_operation(lastid, 2)) {
-        if (LF_ISSET(SCHEMA_DROP_CHECK))
-            check_dropped(session, uri);
-        if (LF_ISSET(SCHEMA_CREATE_CHECK))
-            check_one_entry(session, uri2, uri, uri);
-    }
-    if (has_schema_operation(lastid, 3)) {
         /* Value update operation. */
         gen_table_name(uri, sizeof(uri), lastid - 2, threadid);
-        gen_table2_name(uri2, sizeof(uri2), lastid - 2, threadid);
+        gen_updated_value(uri2, sizeof(uri2), lastid - 2, threadid);
         if (LF_ISSET(SCHEMA_VERBOSE))
-            fprintf(stderr, " update %s\n", uri2);
+            fprintf(stderr, " update %s\n", uri);
```

Comment:
Old line 825 is printing the newly generated value. New line 815 is printing the unchanged table and key. It seems you should print the newly generated value. Please change back to:
          fprintf(stderr, " update %s\n", uri2);
If no scripts are analyzing the output it would be better to make it more clear:
         fprintf(stderr, " update %s to %s\n", uri, uri2);

Reply:
Hi Tod, based on what we do in the previous checks and previous code, the print is to indicate which table the operation will be on. Here we are printing that the update will take place on table 1 (uri). (uri2) is generated as a value to check that the first table (uri) and key has the updated value (uri2) in the following line for `check_one_entry.`